cloudflare/resource_cloudflare_load_balancer_monitor_test.go

```diff
@@ -59,6 +59,29 @@ func TestAccCloudflareLoadBalancerMonitor_FullySpecified(t *testing.T) {
 					resource.TestCheckResourceAttr(name, "header.#", "1"),
 					resource.TestCheckResourceAttr(name, "retries", "5"),
 					resource.TestCheckResourceAttr(name, "port", "8080"),
+					resource.TestCheckResourceAttr(name, "expected_body", "dead"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccCloudflareLoadBalancerMonitor_EmptyExpectedBody(t *testing.T) {
+	t.Parallel()
+	var loadBalancerMonitor cloudflare.LoadBalancerMonitor
+	name := "cloudflare_load_balancer_monitor.test"
```

Comment:
For the name, let's swap it to a randomly generated ID to avoid any potential collisions.

```suggestion
      rnd := generateRandomResourceName()
	name := fmt.Sprintf("cloudflare_load_balancer_monitor.%s", rnd)
```

Reply:
fixed in a712f502ef7896e94b15b80c0aa38c0e02e1ab05